server/datastore/mysql/migrations/tables/20240829170033_AddVPPTokenIDToVppAppsTeams.go

```diff
@@ -14,7 +16,11 @@ func Up_20240829170033(tx *sql.Tx) error {
 ALTER TABLE vpp_apps_teams
 	ADD COLUMN vpp_token_id int(10) UNSIGNED NOT NULL`
 
-	stmtAssociate := `UPDATE vpp_apps_teams SET vpp_token_id = (SELECT id FROM vpp_tokens LIMIT 1)`
+	stmtFindToken := `SELECT id FROM vpp_tokens LIMIT 1`
```

Comment:
Looks like the linter doesn't like this name

Reply:
lol, so dumb! fixed, thanks!